### PR TITLE
Automatically enable HiDPI scaling under Linux

### PR DESCRIPTION
**Pull request by pchote**
~Depends on #17581.~

### CODE REVIEW

**Comment by pchote** [comment]
Rebased.

File: OpenRA.Platforms.Default/Sdl2PlatformWindow.cs
Comment (abcdefg30) [inline OpenRA.Platforms.Default/Sdl2PlatformWindow.cs]: Nothing major, but this could simplify to `if (line.StartsWith("Xft.dpi") && int.TryParse(line.Substring(8), out dpi))`.

File: OpenRA.Platforms.Default/Sdl2PlatformWindow.cs
Comment (abcdefg30) [inline OpenRA.Platforms.Default/Sdl2PlatformWindow.cs]: Again not sure what the purpose of this check is.

**Review by Mailaender — DISMISSED** [review]
Works fine under Linux/KDE/X11.

**Comment by Mailaender** [comment]
Also fixes the local error `OPENRA_DISPLAY_SCALE=1.5 ./launch-game.sh` :+1:

**Comment by pchote** [comment]
Updated.

**Comment by pchote** [comment]
We don't have any other Linux devs to expect a review from, so I will merge this tomorrow if there are no further comments.

**Review by abcdefg30 — APPROVED** [review]

**Comment by abcdefg30** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/931776caf142270d738c06ef56d40711b4c5f9c0...bdfd6eafef6d8a93088227654f1bf671fc49c2f8)

**Pull request merged**
